frontend/src/components/StatsCard.js

```diff
@@ -0,0 +1,207 @@
+import React, { useEffect, useState } from "react";
+import { Stack, Text, chakra, Box, SimpleGrid, Link } from "@chakra-ui/react";
+import { TriangleDownIcon, TriangleUpIcon } from "@chakra-ui/icons";
+import useNFTs from "../core/hooks/useNFTs";
+import web3 from "web3";
+
+const TIME_PERIOD = {
+  current: 0,
+  previous: 1,
+};
+
+const isNumberNotZero = (str) => {
+  if (isNaN(Number(str) || Number(str) == 0)) {
+    return false;
+  } else {
+    return true;
+  }
+};
+
+const getEthValue = (string) => {
+  const ether = web3.utils.fromWei(string, "ether");
+  return nFormatter(ether, 3);
+};
+
+const nFormatter = (num, digits) => {
+  const lookup = [
+    { value: 1, symbol: "" },
+    { value: 1e3, symbol: "k" },
+    { value: 1e6, symbol: "M" },
+    { value: 1e9, symbol: "G" },
+    { value: 1e12, symbol: "T" },
+    { value: 1e15, symbol: "P" },
+    { value: 1e18, symbol: "E" },
+  ];
+  const rx = /\.0+$|(\.[0-9]*[1-9])0+$/;
+  var item = lookup
+    .slice()
+    .reverse()
```

Comment:
Why not just create them in the correct order? (and put a comment indicating that order matters).

Reply:
because then it won't be `self-documenting`?